INCLUDED-MODS.md

```diff
@@ -0,0 +1,100 @@
+## Included mods
+
+Here you can find the mod comparison table between the currently supported Minecraft versions. Some mods may have been obsoleted or superseded by better alternatives in the newer versions.
+
+The selected mods are inspired, but not limited by the [list of OptiFine alternatives](https://lambdaurora.dev/optifine_alternatives). The modpack also [improves the default settings](https://fabulously-optimized.gitbook.io/modpack/readme/changed-options).
+
+### Smooth
```

Comment:
of course there is

Reply:
> of course there is

Yes.
![unknown](https://user-images.githubusercontent.com/77157639/169116317-f3f97a95-7dc3-4843-83d9-aab829479409.png)

Reply:
https://github.com/RaptaG/FO-greek-translations-and-more/commit/6a9c3f65692b4aa37659ac8d3206973121776362